lib/membrane_live_audio_mixer.ex

```diff
@@ -82,23 +98,43 @@ defmodule Membrane.LiveAudioMixer do
 
   @impl true
   def handle_init(_ctx, options) do
-    if options.native_mixer && !options.prevent_clipping do
-      raise("Invalid element options, for native mixer only clipping preventing one is available")
-    else
-      state =
-        options
-        |> Map.from_struct()
-        |> Map.put(:mixer_state, nil)
-        |> Map.put(:live_queue, nil)
-        |> Map.put(:stream_format, nil)
-        |> Map.put(:end_of_stream?, false)
-        |> Map.put(:started?, false)
-        |> Map.put(:eos_scheduled, false)
-
-      {[], state}
+    cond do
+      options.native_mixer && !options.prevent_clipping ->
+        raise "Invalid element options, for native mixer only clipping preventing one is available"
+
+      options.latency == nil and options.stream_format == nil ->
+        raise "Stream format has to be set to start mixing manually"
```

Comment:
Let's add in latency option description what `mixing manually` means, since that term is used in this exception.